nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/client/ESRestClientFactory.java

```diff
@@ -152,9 +150,12 @@ protected SSLContext getSslContext(ElasticSearchClientConfig config) {
         try {
             Path keyStorePath = Paths.get(config.getOption(KEYSTORE_PATH_OPT));
             String keyStorePass = config.getOption(KEYSTORE_PASSWORD_OPT);
-            KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+            String keyStoreOption = config.getOption(KEYSTORE_TYPE_OPT);
+            String keyStoreType = StringUtils.isBlank(keyStoreOption) ? KeyStore.getDefaultType() : keyStoreOption;
```

Comment:
Simplify to `StringUtils.defaultIfBlank(config.getOption(KEYSTORE_TYPE_OPT), KeyStore.getDefaultType())`

Reply:
nice